diff --git a/src/interpreter/Engine.cpp b/src/interpreter/Engine.cpp
index 39c06afbc..02cfb562c 100644
--- a/src/interpreter/Engine.cpp
+++ b/src/interpreter/Engine.cpp
@@ -605,8 +605,8 @@ RamDomain Engine::execute(const Node* node, Context& ctxt) {
                     try {
                         sub_str = str.substr(idx, len);
                     } catch (...) {
-                        std::cerr << "warning: wrong index position provided by substr(\"";
-                        std::cerr << str << "\"," << (int32_t)idx << "," << (int32_t)len << ") functor.\n";
+                        // std::cerr << "warning: wrong index position provided by substr(\"";
+                        // std::cerr << str << "\"," << (int32_t)idx << "," << (int32_t)len << ") functor.\n";
                     }
                     return getSymbolTable().encode(sub_str);
                 }
diff --git a/src/synthesiser/Synthesiser.cpp b/src/synthesiser/Synthesiser.cpp
index 19c27c2f6..1d618da5e 100644
--- a/src/synthesiser/Synthesiser.cpp
+++ b/src/synthesiser/Synthesiser.cpp
@@ -2422,9 +2422,9 @@ void Synthesiser::generateCode(std::ostream& os, const std::string& id, bool& wi
           "len) {\n";
     os << "   std::string result; \n";
     os << "   try { result = str.substr(idx,len); } catch(...) { \n";
-    os << "     std::cerr << \"warning: wrong index position provided by substr(\\\"\";\n";
-    os << "     std::cerr << str << \"\\\",\" << (int32_t)idx << \",\" << (int32_t)len << \") "
-          "functor.\\n\";\n";
+    // os << "     std::cerr << \"warning: wrong index position provided by substr(\\\"\";\n";
+    // os << "     std::cerr << str << \"\\\",\" << (int32_t)idx << \",\" << (int32_t)len << \") "
+    //       "functor.\\n\";\n";
     os << "   } return result;\n";
     os << "}\n";
 
